torchtitan/experiments/llama4/model/args.py

```diff
@@ -47,6 +47,7 @@ class TransformerModelArgs(BaseModelArgs):
     interleave_moe_layer_step: int = 2
     # token-choice
     top_k: int = 1
+    use_fused_moe_impl: bool = True  # fused or for-loop
```

Comment:
I don't think 'fused' is an accurate description for the user...sounds like the whole MoE layer is fused. 
use_grouped_gemm is clearer.